Scripts/build.sh

```diff
@@ -1,2 +1,20 @@
-echo "xcodebuild build -workspace $TRAVIS_XCODE_WORKSPACE -scheme $TRAVIS_XCODE_SCHEME -sdk $TRAVIS_XCODE_SDK"
-xctool build test -workspace "$TRAVIS_XCODE_WORKSPACE" -scheme "$TRAVIS_XCODE_SCHEME" -sdk "$TRAVIS_XCODE_SDK"
+#checking if xcpretty is available to use
+command -v xcpretty >/dev/null
+if [ $? -eq 1 ]; then
+echo >&2 "xcpretty not found don't use it."
```

Comment:
It will exit immediately without building. I can put an extra if to built it without xcpretty or install it automattically from the web.